impl/src/main/java/org/ehcache/internal/store/heap/OnHeapStore.java

```diff
@@ -939,6 +928,17 @@ private void setAccessTimeAndExpiry(K key, OnHeapValueHolder<V> valueHolder, lon
     return makeValue(value, now, expirationTime, this.valueCopier);
   }
 
+  private OnHeapValueHolder<V> importValueFromLowerTier(K key, ValueHolder<V> valueHolder, long now) {
+    V realValue = valueHolder.value();
+    Duration expiration = expiry.getExpiryForAccess(key, realValue);
+    if(valueCopier instanceof SerializingCopier) {
+      // TODO Pass a copy of the byteBuffer from lower tier and pass it instead of realValue
```

Comment:
Let me remove it - it may remain a possibility - saving a _serialization_ but at the cost of making the authority aware that the on heap tier is serialiazing ... so not sure at all we want that. I'll add a note instead on the issue for that.